http/client.go

```diff
@@ -0,0 +1,78 @@
+// Package http provides an easy way to construct an http client with custom certificates and customizable timeout.
+// If you need to customize other attributes you can use the golang http package. https://golang.org/pkg/net/http/
+package http
+
+import (
+	"crypto/tls"
+	"crypto/x509"
+	"errors"
+	"io/ioutil"
+	"net/http"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+// ClientArguments are meant to be used as flags from a custom integrations. With this you could
+// send this arguments from the command line.
+type ClientArguments struct {
+	HTTPCaBundleFile string        `default: "" help: "Name of the certificate file"`
+	HTTPCaBundleDir  string        `default: "" help: "Path where the certificate exists"`
+	HTTPTimeout      time.Duration `default:30 help: "Client http timeout"`
```

Comment:
Hey, 
have you test this? I think `time.Duration` is nanoseconds by default: https://golang.org/pkg/time/#Duration

Reply:
When setting the timeout we change the units multiplying it by Time.seconds.

I'll add a description in the help.